extension/intents/scroll/scrollHelper.js

```diff
@@ -8,6 +8,9 @@ function getScrollParent(node) {
   }
 
   if (node.scrollHeight > node.clientHeight) {
+    if (node.scrollHeight === node.parentNode.scrollHeight) {
+      return node.parentNode;
+    }
```

Comment:
What does this do? I'm a little confused here... the first test sees if the node can be scrolled (node.scrollHeight > node.clientHeight), but the second test... sees if the entire height of the current node (including portions scrolled off) is equal to the entire height of the parent node (irregardless of the rendered size of the parent node) and then uses that...

Also I wonder, if this is the right test (even though I don't understand it) why just return the parentNode? Why not return the further ancestor for which this is true?

Reply:
While looking for the nearest scrollable element. On these sites mentioned the html and body's height is set to a 100%. When I checked the styling on these sites I noticed that even though the body has a style of `overflow-y: scroll`, the the `html` element gets the scrollable overflow instead. But the scrollable element gotten from the function is the `body` element which does not scroll. I initially made the test to check for the body element but I felt it was not generic enough. In these scenarios the 'body` and `html` scrollable height are also equal. So I leaned towards using that instead

Reply:
OK, I've noticed it before but hadn't understood what quite happened. I think this body/html fix is more specific and probably more correct as a result.

Reply:
I am glad this worked. Thank you very much